src/app/static/src/app/components/downloadResults/DownloadResults.vue

```diff
@@ -31,7 +31,7 @@
                 <div id="uploading" v-if="uploading" class="d-flex align-items-end">
                     <loading-spinner size="xs"></loading-spinner>
                     <div class="d-flex align-items-center height-40 ml-2'">
-                        <span v-translate="'uploading'"></span>
+                        <span v-html="uploadingStatus"></span>
```

Comment:
I don't think this needs to be `v-html` as uploadingStatus is not a raw html string. Can just be `{{uploadingStatus}}` inside the span.